src/sql/workbench/contrib/dashboard/browser/core/dashboardPage.css

```diff
@@ -63,3 +63,23 @@ dashboard-page .editor-toolbar {
 	border-bottom-style: solid;
 	padding: 3px 0;
 }
+
+.dashboardEditor breadcrumb .chevron-right.codicon {
+	background-size: 8px;
+}
+
+.vs-dark .dashboardEditor breadcrumb,
+.hc-black .dashboardEditor breadcrumb {
+	font-size: 13px;
```

Comment:
i think this is not necessary, you get this from the other more generic selector down below